CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Upcoming
+* Add yarn support
```

Comment:
is yarn internal business of the library or in other words, does it affect api surface of the library?

Reply:
i would also add `isomorphic-fetch` unlocking

Reply:
Not the library per se, only dev experience.